bootstrap.lic

```diff
@@ -3,9 +3,29 @@
   Documentation: https://elanthipedia.play.net/Lich_script_development#bootstrap
 =end
 
-class_defs = { 'equipmanager' => :EquipmentManager, 'common' => :DRC, 'common-travel' => :DRCT, 'common-crafting' => :DRCC, 'common-summoning' => :DRCS, 'common-money' => :DRCM, 'common-arcana' => :DRCA, 'common-items' => :DRCI, 'common-healing' => :DRCH, 'common-validation' => :CharacterValidator, 'common-moonmage' => :DRCMM, 'common-theurgy' => :DRCTH }
+$BOOTSTRAP_VERSION = '1.0.1'
```

Comment:
Added this; recently with the upgrades to `drinfomon` and `dependency` we've been tracking individual script versions to help us with testing to confirm which version of the script someone is running

Reply:
Thanks. I wrote a test script to regularly verify versions and dump drinfomon's collected data to check for failures, so this helps verify.